src/__tests__/request-test.js

```diff
@@ -71,12 +71,18 @@ describe("request", () => {
       request("http://localhost").subscribe(observer);
 
       const connectionMock = XHRConnection.mock.instances[0];
+      connectionMock.xhr = { status: 201 };
       connectionMock.response = "some text";
 
-      const connectionEventMock = { target: connectionMock };
+      const connectionEventMock = {
+        target: connectionMock
+      };
       connectionMock.__emit(ConnectionEvent.COMPLETE, connectionEventMock);
 
-      expect(observer.next).toHaveBeenCalledWith("some text");
+      expect(observer.next).toHaveBeenCalledWith({
+        response: "some text",
```

Comment:
Can't we reference `connectionMock.response` here? and `connectionMock.xhr.status` in the next line? Not part of this PR a general question.